### ISSUE TITLE
Files generated with html-critical-webpack-plugin are not compressed by compression-webpack-plugin

### ISSUE THREAD

**Issue by Focus3D** [anthonygore/html-critical-webpack-plugin#21]
Hi,
using "html-critical-webpack-plugin": "^2.0.0", with "compression-webpack-plugin": "^1.1.11"
Files generated by this plugin are not taken in consideration by compression-webpack-plugin.

**Comment by anthonygore** [comment]
Can you make a reproduction of the problem demonstrating how this is an
issue with this plugin?

**Comment by Focus3D** [comment]
Thank you for prompt reaction and as requested, you can check the problem at [https://github.com/Focus3D/test-webpack-criticalcss](https://github.com/Focus3D/test-webpack-criticalcss).
As you will see, html and css generated by html-critical-webpack-plugin are not compressed neither by gzip or brotli plugin.

My env : windows 7 64bits; node v.9.9.0; npm v.6.0.1
Thank you.

**Comment by Focus3D** [comment]
@thescientist13 
Thank you for the heads up, I get it know, so ending up using a script at the end of the build to compress only html and css (removed html and css from webpack compression).
Thank you for providing the info.

**Comment by thescientist13** [comment]
Yeah, it might make sense to update the README here perhaps, to communicate that this plugin functions in more of "post build" kind of way, since it doesn't operate on the actual dependency graph / compilation itself, as it just calls out to the **critical** CLI, which itself just reads / writes to the filesystem.

**Comment by thescientist13** [comment]
updated the documentation in PR #25 .

**Comment by anthonygore** [comment]
I'll close this since the question has been answered, but we could consider making this plugin part of the chain in the future.

**Comment by thescientist13** [comment]
Opened #27 to keep the conversation going and hash out technical design / implementation

**Comment by arvindanta** [comment]
Hi @anthonygore @thescientist13 @Focus3D Any improvements on this?. Am facing the same issue now.
